### ISSUE TITLE
getBars called only once

### ISSUE THREAD

**Issue by axelkeegan** [tradingview/charting-library-tutorial#13]
Greetings,

thank you team for this great tutorial. I managed to switch the timeframe to 1 minute chart successfully, but getBars method is calling only once. Currently only the last bar is receiving the updates, and new bars are not generated.

**Comment by timocov** [comment]
can you provide steps to reproduce please?

**Comment by axelkeegan** [comment]
will provide info later today. sorry for the delay.

**Comment by axelkeegan** [comment]
onReady method added support for 1,5,15 minutes.

```
....
const configurationData = {
    supported_resolutions: ['1D', '1W', '1M'],
    exchanges: [
....
```

Turned intraday to true, in order to enable above resolutions

```
....
   has_intraday: false,
....
```

inside getBars method, changed 

`            const data = await makeApiRequest(`data/histoday?${query}`);
` 
to

`            const data = await makeApiRequest(`data/histominute?${query}`);
`

Basically nothing else was changed from the tutorial steps. The end result is perfectly working 1-5-15 minute TV chart, data is collected successfully. Once TV chart is ready and all bars are drawn, all next changes are drawn only on the last candle, no new candles are drawn. No matter which resolution is used 1-5-15.

Will be happy to understand how i can make it working on 1-5-15 minutes chart.

Another interesting problem is, if we have two separate servers where this tutorial is loaded, if we open them in two separate browser tab, only one of them is working because of 'CORS'. But this is another issue.

Regards,

**Comment by axelkeegan** [comment]
UP @timocov

**Comment by timocov** [comment]
Sorry for late reply.

Can you prepare a repro for that and steps for reproduce after that changes please? A repo or a git diff with changes so I can apply them and see what's going on?

> Another interesting problem is, if we have two separate servers where this tutorial is loaded, if we open them in two separate browser tab, only one of them is working because of 'CORS'. But this is another issue.

Not sure how 2 tabs are related to the CORS problem.

**Comment by axelkeegan** [comment]
No problem for the late reply, i suppose all of us are busy enough already.

Will be hard to upload the project to repo since already deleted the project due to debug process stuck. I believe will be much easier, if you are able to share some guidance how to convert your tutorial to 1-5-15 minutes chart configuration.

Regards,

**Comment by timocov** [comment]
Ok, I marked it as feature request to add intraday resolutions.

**Comment by axelkeegan** [comment]
Highly appreciate it. Any expected ETA on it ?

Regards,

**Comment by axelkeegan** [comment]
UP @timocov

**Comment by timocov** [comment]
No, we don't have any ETA for that. If somebody wants to fix it - PRs are welcomed.

**Comment by axelkeegan** [comment]
Some guiding lines will be appreciated ...

**Comment by EzyBot** [comment]
: D ETAs are for professional companies, after you checked out the guides from them, you know there are not

**Comment by vladyosifov** [comment]
one year later no resolution ? is that hard to be converted to add intraday resolutions ?
